iceoryx_hoofs/vocabulary/include/iox/string.hpp

```diff
@@ -545,7 +545,7 @@ class string final
 ///
 /// @return the stream output of the fixed string
 template <uint64_t Capacity>
-std::ostream& operator<<(std::ostream& stream, const string<Capacity>& str) noexcept;
+std::ostream& operator<<(std::ostream& stream, const string<Capacity>& str) noexcept(false);
```

Comment:
Because then there will be a warning that the specifier is missing :grimacing: 

I'd like to avoid adding suppressions when it can be fixed so easily without affecting the code using it.

Reply:
@elBoberido I understand that. But if it's only those to occurrences, I would prefer a suppression and a `@todo`. We will work on this anyway with `v4.0`.

Reply:
I am fine with either way. But I think it is in general better to adjust the code so that no suppression is required.

@mossmaurice we can also find those code snippets again when we just grep for `noexcept(false)` which we most likely have to do either way since I would bet that not every `noexcept(false)` has a todo.